app/Http/Controllers/Api/V1/QuestionBankController.php

```diff
@@ -1580,6 +1582,123 @@ private function handleChildren(QuestionBankVersion $questionVersion, array $inp
         }
     }
 
+    private function patchChildren(QuestionBankVersion $questionVersion, array $input, array $jwtUser, QuestionBank $parent): void
```

Comment:
is .... long

Reply:
I split out create and update child methods, which may be better.